app/src/main/java/com/lucascabral/marvelsuperheroes/presenter/viewmodel/AllCharactersViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.lucascabral.marvelsuperheroes.presenter.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.PagingData
+import androidx.paging.cachedIn
+import com.lucascabral.marvelsuperheroes.network.MarvelService
+import com.lucascabral.marvelsuperheroes.network.model.Character
+import com.lucascabral.marvelsuperheroes.repository.paging.CharacterPagingSource
+import kotlinx.coroutines.flow.Flow
+import org.koin.core.KoinComponent
+import org.koin.core.inject
+
+
+class AllCharactersViewModel: ViewModel(), KoinComponent {
+
+     private val apiService: MarvelService by inject()
+
+     fun getListData(): Flow<PagingData<Character>> {
+          return Pager(config = PagingConfig(pageSize = 20, maxSize = 200),
```

Comment:
Esses valores fixos aqui seria interessante criar variáveis constantes pra eles, pra evitar [magic numbers](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad). Poderiam ficar dentro de um `companion object` dentro dessa `ViewModel` mesmo.

Reply:
Feito.